modules/backup-and-restore/pages/point-in-time-restore.adoc

```diff
@@ -0,0 +1,76 @@
+= Point-in-Time Restore
+
+The introduction of Point-in-Time Restore in version 4.1.0 is designed to enable users to restore the database to a previous time point, they can provide a timestamp and restore to it using previous made backups, even though no backup has been conducted at exactly that time point.
```

Comment:
Let's improve this introduction for clarity purpose:
Point-in-Time Restore, introduced in version 4.1.0, allows users to restore the database to a specific previous time point. By providing a timestamp, users can restore the database using previous backups, even if no backup was created exactly at that time.

Reply:
Done